src/config/backpack/crud.php

```diff
@@ -602,4 +602,19 @@
         // "zu" => "Zulu"
     ],
 
+    'view_namespaces' => [
+        'buttons' => [
+            'crud::buttons', // falls back to 'resources/views/vendor/backpack/crud/buttons'
+        ],
+        'columns' => [
+            'crud::columns', // falls back to 'resources/views/vendor/backpack/crud/columns'
+        ],
+        'fields' => [
+            'crud::fields', // falls back to 'resources/views/vendor/backpack/crud/fields'
+        ],
+        'filters' => [
+            'crud::filters', // falls back to 'resources/views/vendor/backpack/crud/filters'
+        ],
+    ],
+
```

Comment:
Talked to @promatik - maybe we should move this to BackpackServiceProvider - that way the devs don't even see this.

Reply:
Nah... It wouldn't be clear enough. I say let's keep it like this for now.